src/include/platform_support.h

```diff
@@ -35,6 +35,7 @@ void platform_timeout_set(platform_timeout *t, uint32_t ms);
 bool platform_timeout_is_expired(platform_timeout *t);
 void platform_delay(uint32_t ms);
 
+extern bool connect_assert_srst;
```

Comment:
this doesn't seem to be used for anything

Reply:
It is needed for hosted and command.c. Have to think aboyur that